torch/csrc/jit/passes/onnx/helper.h

```diff
@@ -14,6 +14,8 @@ static const int OPSET_VERSION_10 = 10;
 static const int OPSET_VERSION_11 = 11;
 static const int OPSET_VERSION_12 = 12;
 static const int OPSET_VERSION_13 = 13;
+static const int OPSET_VERSION_14 = 14;
+static const int OPSET_VERSION_15 = 15;
```

Comment:
BE question: shouldn't it be enum?
```suggestion
enum {
  OPSET_VERSION_11 = 11,
  OPSET_VERSION_12,
  OPSET_VERSION_13,
  OPSET_VERSION_14,
  OPSET_VERSION_15,
};
```